static/channel.html

```diff
@@ -88,6 +88,8 @@ <h3>
     </main>
     <footer>LavinMQ is open source and developed by <a href="https://www.84codes.com" target="_blank"><img class="logo" src="/img/logo-84codes.svg"></a></footer>
     <script src="/js/lib/chart.js"></script>
+    <script src="/js/lib/moment.js"></script>
```

Comment:
use the ESM variants of these libraries and `import` them in our `chart.js`, that way you don't need to include them in each html file.